charts/vela-core/templates/addons/terraform-provider-alibaba.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+data:
+  initializer: |
+    apiVersion: core.oam.dev/v1beta1
+    kind: Initializer
+    metadata:
+      annotations:
+        addons.oam.dev/description: terraform provider (alibaba)
+        addons.oam.dev/name: terraform/provider-alibaba
+      name: terraform-provider-alibaba
+      namespace: vela-system
+    spec:
+      appTemplate:
+        spec:
+          components:
+          - name: alibaba-account-creds
+            properties:
+              apiVersion: v1
+              kind: Secret
+              metadata:
+                name: alibaba-provider-secret
+                namespace: vela-system
+              stringData:
+                credentials: |
+                  accessKeyID: [[ index .Args "ak" ]]
+                  accessKeySecret: [[ index .Args "sk" ]]
+                  securityToken: [[ index .Args "sts" ]]
+              type: Opaque
+            type: raw
+        status:
+          rollout:
+            batchRollingState: ""
+            currentBatch: 0
+            lastTargetAppRevision: ""
+            rollingState: ""
+            upgradedReadyReplicas: 0
+            upgradedReplicas: 0
+    status:
+      observedGeneration: 0
```

Comment:
Are there fields necessary?

Reply:
generated when Marshual application to yaml, need deleted?

Reply:
Then feel free to delete it.